### PR TITLE
Create liege.ruleset

### PR DESCRIPTION
**Pull request by jmleroy**
800 years of history deserved a ruleset

### CODE REVIEW

File: data/nation/liege.ruleset [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+flag="liege"
+flag_alt = "belgium"
+style = "Celtic"
+
+init_techs=""
+init_buildings=""
+init_units=""
+conflicts_with="belgian", "walloon", "flemish"
+civilwar_nations="belgian", "walloon", "flemish"
```
Comment (nicoe) [inline data/nation/liege.ruleset:42]: I would use "grignoux" and "chiroux"

File: data/nation/liege.ruleset [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+; 
+cities =
```
Comment (nicoe) [inline data/nation/liege.ruleset:45]: i don't know how you decided the order of the cities but I think that Saint-Hubert should be one of the first given the importance it had for one of the first Prince-Bishop

File: data/nation/liege.ruleset [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+flag="liege"
+flag_alt = "belgium"
+style = "Celtic"
+
+init_techs=""
+init_buildings=""
+init_units=""
+conflicts_with="belgian", "walloon", "flemish"
```
Comment (nicoe) [inline data/nation/liege.ruleset:41]: Is there any civilization of the Duke of Burgundy, in such case they should be there!

File: data/nation/liege.ruleset [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+ruler_titles = {
+ "government",      "male_title",           "female_title"
+ "Monarchy",        _("Prince %s"),         _("?female:Princess %s")
+ "Democracy",       _("Prime Minister %s"), _("?female:Prime Ministress %s")
+ "Fundamentalism",  _("Prince-Bishop %s"),  _("Mother Superior %s")
+ "Republic",        _("Regent %s"),         _("Regent %s")
+}
+
+flag="liege"
```
Comment (nicoe) [inline data/nation/liege.ruleset:34]: I think you need to define a flag. I am not sure though.

File: data/nation/liege.ruleset [hunk 5]
```diff
@@ -0,0 +1,78 @@
+[nation_liege]
+
+name=_("Liegean")
+plural=_("?plural:Liegeans")
+groups="Modern", "European"
+legend = _("Liege was constituted in ecclesiastic principalty in 985,\
+ when Bishop Notger was granted the control of several cities in current Belgium.\
+ As a federated state of the Holy Roman Empire, it was characterized\
+ with large civil liberties for which liegean struggled strongly for.\
```
Comment (nicoe) [inline data/nation/liege.ruleset:9]: I am not sure it's the right demonym

File: data/nation/liege.ruleset [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+leaders = {
+ "name",                   "sex"
+ "Notger",                 "Male"
+ "Wazon",                  "Male"
+ "Albert de Cuyck",        "Male"
+ "Érard de la Marck",      "Male"
+ "Jean-Nicolas Bassenge",  "Male"
+ "Joseph Bologne",         "Male"
+ "Julienne de Cornillon",  "Female"
+ "Théroigne de Méricourt", "Female"
```
Comment (nicoe) [inline data/nation/liege.ruleset:23]: You could also put some more recent figures like "Ferdinand Piercot", "Émile Digneffe" or "Gustave Kleyer".

File: data/nation/liege.ruleset (hunk 2 above)
Comment (jmleroy) [reply data/nation/liege.ruleset:45]: It's the order in which these cities were granted the status of "Bonne Ville" according to wikipedia. Plus Maastricht and Bouillon, which were associated cities but not strictly under Liege's sovereignity, and a few extra cities (starting from Seraing).
Also, I noticed that I forgot to add Franchimont among the extra cities. Their sacrifice against Burgundy still resonates.

File: data/nation/liege.ruleset (hunk 3 above)
Comment (jmleroy) [reply data/nation/liege.ruleset:41]: Absolutely! Charles the Bold to the wolves!

File: data/nation/liege.ruleset (hunk 4 above)
Comment (jmleroy) [reply data/nation/liege.ruleset:34]: I don't know how. However, it's pretty simple : https://en.wikipedia.org/wiki/Republic_of_Li%C3%A8ge#/media/File:LuikVlag.svg

File: data/nation/liege.ruleset (hunk 5 above)
Comment (jmleroy) [reply data/nation/liege.ruleset:9]: I can't find it on the interwebs, and "liegian" is used for somewhere else.

File: data/nation/liege.ruleset (hunk 6 above)
Comment (jmleroy) [reply data/nation/liege.ruleset:23]: Too many males already

Comment (jmleroy) [reply data/nation/liege.ruleset:23]: For the record: 
Julienne de Cornillon : https://en.wikipedia.org/wiki/Juliana_of_Li%C3%A8ge
Théroigne de Méricourt : https://en.wikipedia.org/wiki/Theroigne_de_Mericourt

**Comment by cazfi** [comment]
See http://www.freeciv.org/wiki/How_to_Contribute about how to contribute, and doc/README.nations about what is required from a nation submission to be acceptable.

**Comment by cazfi** [comment]
Public Domain svg flag on Wikimedia Commons: https://commons.wikimedia.org/wiki/File:LuikVlag.svg

**Comment by cazfi** [comment]
Opened a ticket about handling this -> https://osdn.net/projects/freeciv/ticket/46553

**Comment by cazfi** [comment]
Pushed in outside gihub UI.

**Pull request closed without merging**